src/content/docs/zh-cn/reference/error-reference.mdx

```diff
@@ -71,7 +71,6 @@ githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/
 - [**CantRenderPage**](/zh-cn/reference/errors/cant-render-page/)<br/>Astro can't render the route.
 - [**UnhandledRejection**](/zh-cn/reference/errors/unhandled-rejection/)<br/>Unhandled rejection
 - [**i18nNotEnabled**](/zh-cn/reference/errors/i18n-not-enabled/)<br/>i18n Not Enabled
-- [**RouteNotFound**](/zh-cn/reference/errors/route-not-found/)<br/>Route not found.
```

Comment:
The `RouteNotFound` error should still exist under Astro Errors, it was just a mistake that it was added in another category!

Reply:
OK.